profiles/package.mask/text.xml

```diff
@@ -14,6 +14,12 @@ have a comment detailing the specific reason for the mask. The format
 of the <c>package.mask</c> file is described in <c>man portage</c>.
 </p>
 
+<p>
+Entries should be added chronologically <d/> that is, newer entries
+should be placed towards the top of the file, underneath any initial
+header comment block.
```

Comment:
I'd write "Entries are added chronologically ..." for a somewhat stronger statement.
No blocker for this PR though, it's also fine if you prefer the current wording.

Reply:
Works for me, I'll change it, thanks!